src/schemas/validateLoginSchema.ts

```diff
@@ -0,0 +1,14 @@
+import * as yup from "yup";
+
+export const validateLoginSchema = yup.object().shape({
+  username: yup
+    .string()
+    .required("Name is required")
+    .min(5, "Min length is 5 characters")
```

Comment:
There are people with short names of only 2 characters. Or really long ones.

We should use a min of 1 and a maximum of 50 maybe.

Reply:
Just changed this messages for the next commit thank you